### PR TITLE
Updated localadmin and region records in Catalonia

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1613.

- Updates the geometries of localadmin records in Catalonia, Spain and stores existing geometries in alt files
- Adds source properties to each localadmin record
- Deprecates one localadmin record, flagging it as not current
- Updates the four regions within Catalonia and stores existing geometries in alt files
- Adds source properties to each region record

This PR requires PIP work (and validation of geometries) before merge.

Source: https://github.com/whosonfirst/whosonfirst-sources/pull/150

### CODE REVIEW

**Comment by nvkelso** [comment]
(Can proceed with PIP.)

**Comment by stepps00** [comment]
To do:

- [ ] Store concordance value
- [ ] Update source prefix in properties
- [ ] PIP
- [ ] Validate changes and geometries

**Comment by stepps00** [comment]
@nvkelso - most recent commits store concordance values, update source prefix, add PIP work to update hierarchies, and validate GeoJSON/geometries. This PR is ready for merge.

File: data/404/330/971/404330971.geojson [hunk 1]
```diff
@@ -72,12 +72,13 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
+        "esp-icgc:la_lc":8041,
```
Comment (nvkelso) [inline data/404/330/971/404330971.geojson]: Missing a leading `0`, please add this back in a string (for all records).

Please also file followup issue for gleaning more Spanish "FIPS" codes from the QS props (similar to how you've done here).

Comment (stepps00) [reply data/404/330/971/404330971.geojson]: Fixed in 5c7c225

**Pull request merged**